packages/cli/src/commonlib/cacheAccess.ts

```diff
@@ -201,3 +202,54 @@ export async function loadAccountId(accountName: string) {
 
   return undefined;
 }
+
+export async function saveAzureSP(
+  clientId: string,
+  secret: string,
+  tenantId: string
+): Promise<void> {
+  await fs.ensureDir(cacheDir);
+  const azureSPCrypto = new AccountCrypto("azure-sp");
```

Comment:
Use `this.accountCrypto`, do not create another AccountCrypto.

Reply:
Thanks for the advice, I have changed ```AzureSpCypto``` method to static method.